docs/user-guide/compare-options.md

```diff
@@ -2,9 +2,10 @@
 
 ## Ignoring Resources That Are Extraneous
 
->v1.1
+!!! note
+    available in Argo CD starting from version 1.1
```

Comment:
Please remove the version comments entirely. They're leftover from before we had a versioned docs site. :-)